Office 365/o365/ingest/parser.yml

```diff
@@ -116,7 +253,36 @@ stages:
           office365.context.correlation.id: "{{json_event.message.AppAccessContext.CorrelationId}}"
         filter: '{{json_event.message.get("AppAccessContext") != None}}'
 
-  parse_exchange:
+  parse_exchange_admin:
+    actions:
+      - translate:
+        dictionary:
+          "True": "success"
+          "False": "failure"
+        mapping:
+          json_event.message.ResultStatus: action.outcome
+      - set:
+          office365.context.aad_session_id: "{{json_event.message.SessionId}}"
+      - set:
+          office365.exchange_admin.parameters: >
+            {
+              {% for parameter in json_event.message.Parameters %}
+                {% if parameter.Name not in ["AlwaysDeleteOutlookRulesBlob", "Force"] %}
+                  "{{ parameter.Name }}": "{{ parameter.Value }}",
+                {% endif %}
+              {% endfor %}
+            }
+        filter: '{{json_event.message.get("Parameters") != None}}'
```

Comment:
We dislike unbounded dictionary. May cause issue for the indexation.

Reply:
I could explicitely set the list of the Parameters that we allow.
How many can we handle?

Reply:
I think that we can do most of the rules that we want for now with those 7 parameters:
* Name
* MoveToFolder
* ForwardTo
* RedirectTo
* DeleteMessage
* StopProcessingRules
* SubjectOrBodyContainsWords

I'll make the change.